_sass/_base.scss

```diff
@@ -0,0 +1,26 @@
+body {
+  background: $color-light;
+  background-image: url("#{$assets-path}f8a323a7-image-background-paper.png");
+}
+
+h2 {
+  margin-bottom: 2.5rem;
+}
+
+.p-list--get-started {
+  font-size: 1.4375rem;
+}
+
+.p-button--neutral-icon {
+  align-items: center;
+  background: none;
+  display: flex;
+  padding: .3rem 1.5rem !important;
```

Comment:
Instead of using `!important` can you remove the `p-button--neutral` class and extend it here.
```scss
.p-button--neutral-icon {
  @include button-neutral;
  ...
}
```